CMakeLists.txt

```diff
@@ -53,7 +56,11 @@ option(compileOption_C "passes a string to the command line of the C compiler" O
 option(compileOption_CXX "passes a string to the command line of the C++ compiler" OFF)
 
 # Start of variables used during install
-set (LIB_INSTALL_DIR lib CACHE PATH "Library object file directory")
+if (CMAKE_INSTALL_LIBDIR)
```

Comment:
This variable will always be set, right? If you include GNUInstallDirs. And the default is already "lib" I think.